src/component/contents/DraftEditorLeaf.react.js

```diff
@@ -142,9 +142,15 @@ class DraftEditorLeaf extends React.Component<Props> {
     }
 
     const {customStyleMap, customStyleFn, offsetKey, styleSet} = this.props;
+    const classNames = [];
     let styleObj = styleSet.reduce((map, styleName) => {
       const mergedStyles = {};
-      const style = customStyleMap[styleName];
+      let style;
+      if (typeof customStyleMap[styleName] === 'string') {
```

Comment:
Maybe would be better to add a new prop `customCssClassMap`. This way you don't need this check.

Reply:
I did this approach here https://github.com/facebook/draft-js/pull/2930

Reply:
Your version looks cleaner. I like it)